### PR TITLE
Fix thread stack size on some embedded devices.

### PR DESCRIPTION
**Pull request by ysc3839**
Code from: https://github.com/wishinlife/SyncY/blob/master/syncy.py#L337-L355

### CODE REVIEW

**Comment by ysc3839** [comment]
I don't know if it's good to check it default. I think we should add a option.

File: src/Config.py [hunk 1]
```diff
@@ -60,6 +60,26 @@ def createArguments(self):
...
+                break
+            except thread.error:
+                thread.stack_size(0)
+                break
+            except RuntimeError:
+                thread.stack_size(0)
+                break
+            except ValueError:
+                if stack_size < 512 * 1024:
+                    stack_size *= 2
```
Comment (cxgreat2014) [inline src/Config.py]: ？？？乘了2之后没有使用这个值呀，有啥意义？

Comment (cxgreat2014) [inline src/Config.py]: 哦哦哦，while True，之后进行下一次了，手机看代码没看全，sorry

**Comment by ysc3839** [comment]
This code is from [SyncY](https://github.com/wishinlife/SyncY), which is licensed under an 'unopen' license. I don't know if it's good to use it's code on a GPL project.

**Pull request closed without merging**